daliuge-engine/test/manager/test_dim.py

```diff
@@ -356,34 +356,36 @@ def test_fullRound(self):
         """
 
         sessionId = "lala"
-        restPort = 8989  # don't interfere with EAGLE default port
-        args = ["--port", str(restPort), "-N", hostname, "-qqq"]
+        nmPort = 8000  # NOTE: can't use any other port yet.
```

Comment:
If this is a default port, shouldn't we be adding it to the DEFAULT_PORTS in `dlg.constants`? Same comment for the 8989 EAGLE port.